### ISSUE TITLE
Can't delete victoria-metrics-k8s-stack

### ISSUE THREAD

**Issue by kevinvirs** [VictoriaMetrics/helm-charts#131]
helm uninstall [release] -n namespace
not fully uninstalled

```
vmagent-vm-victoria-metrics-k8s-stack-5ff5cb58c8-8swvf   2/2     Running            0         3h37m
vmalert-vm-victoria-metrics-k8s-stack-5447c58c65-9kqsb   2/2     Running            0          3h37m
vmsingle-vm-victoria-metrics-k8s-stack-dc675fbc4-rfbns   1/1     Running            0          3h37m
```

can't remove manually

`kubectl delete vmagent -n namespace vm-victoria-metrics-k8s-stack`

not working and can't install again
```

>helm uninstall -n namespace vm
Error: uninstall: Release not loaded: vm: release: not found

>helm upgrade --install -n namespace vm vm/victoria-metrics-k8s-stack
Release "vm" does not exist. Installing it now.
Error: rendered manifests contain a resource that already exists. Unable to continue with install: existing resource conflict: namespace: namespace, name: vm-victoria-metrics-k8s-stack, existing_kind: operator.victoriametrics.com/v1beta1, Kind=VMAgent, new_kind: operator.victoriametrics.com/v1beta1, Kind=VMAgent
```

**Comment by f41gh7** [comment]
Hello, this is bug, need to fix it at helm chart. All crd objects has finalizers, that operator must remove after crd object fully deleted, but helm uninstall removes operator before it finished resources deletion.

 Workaround to remove finalizers:
```yaml
kubectl get vmagents  | grep -v NAME | awk '{print $1}' | xargs -i  kubectl patch vmagent {} -p '{"metadata":{"finalizers":null}}' --type=merge
kubectl get vmalerts  | grep -v NAME | awk '{print $1}' | xargs -i  kubectl patch vmalerts {} -p '{"metadata":{"finalizers":null}}' --type=merge
kubectl get vmsingles  | grep -v NAME | awk '{print $1}' | xargs -i  kubectl patch vmsingles {} -p '{"metadata":{"finalizers":null}}' --type=merge

```

**Comment by kevinvirs** [comment]
```
>kubectl patch vmagent vm-victoria-metrics-k8s-stack -p '{"metadata":{"finalizers":null}}' --type=merge
Error from server: Invalid JSON Patch

>kubectl patch vmsingles vm-victoria-metrics-k8s-stack -p '{"metadata":{"finalizers":null}}' --type=merge
Error from server: Invalid JSON Patch

>kubectl patch vmalert vm-victoria-metrics-k8s-stack -p '{"metadata":{"finalizers":null}}' --type=merge
Error from server: Invalid JSON Patch
```
not working :((

**Comment by f41gh7** [comment]
Hm, try to simply edit object and remove `finalizers` record, something like:
```yaml
kubectl edit vmalert vm-victoria-metrics-k8s-stack
```
 and remove metadata.finalizers field.

**Comment by f41gh7** [comment]
Must be fixed with new stack version `0.2.2`
